app/views/tutoring_session/show.html.erb

```diff
@@ -14,8 +14,8 @@
         Current Users:
       </p>
       <p>
-      <% @tsession.users.each do |user| %>
-        <%= user.first_name %> <%= user.last_name %> 
+      <% @tsession.tutoring_session_users.each do |link| %>
+        <%= link.user.first_name %> <%= link.user.last_name %> | <%= link.link_status %>
```

Comment:
What's the purpose of this change? Does anything benefit by using the join-table?

Reply:
Link status is only viewable through the join table